frontend/admin/src/js/components/poolCandidate/CreatePoolCandidate.tsx

```diff
@@ -255,7 +255,7 @@ export const CreatePoolCandidateForm: React.FunctionComponent<CreatePoolCandidat
         case "new":
           userObject = {
             create: {
-              email: values.email,
+              email: values.email ?? "",
```

Comment:
What's the purpose behind this change?

Reply:
This is because email is now nullable in the inital data but is still non-nullable for CreateUserInput.

Reply:
Ah, I see, its to get around type shenanigans :joy:

Reply:
May I recommend adjusting the FormValues type to better reflect the form, instead of correcting for it here?
```
type FormValues = Pick<
  PoolCandidate,
  | "cmoIdentifier"
  | "expiryDate"
  | "hasDiploma"
  | "hasDisability"
  | "isIndigenous"
  | "isVisibleMinority"
  | "isWoman"
  | "languageAbility"
  | "expectedSalary"
  | "locationPreferences"
  | "acceptedOperationalRequirements"
  | "status"
> &
  Pick<User, "firstName" | "lastName" | "preferredLang" | "telephone"> & {
    email: string; // email is Maybe<string> in User definition, but is required in the form.
    cmoAssets: string[] | undefined;
    expectedClassifications: string[] | undefined;
    pool: string;
    user: string;
    userMode: UserModeValue;
  };
  ```

Reply:
Yeah, this is better, thanks.